samples/wifi/promiscuous/sample.yaml

```diff
@@ -0,0 +1,32 @@
+sample:
+  description: Wi-Fi Promiscuous sample
+    application
+  name: Wi-Fi Promiscuous sample
+tests:
+  sample.nrf7002.promiscuous:
+    build_only: true
+    integration_platforms:
+      - nrf7002dk/nrf5340/cpuapp
+    platform_allow: nrf7002dk/nrf5340/cpuapp
+    tags: ci_build
+  sample.nrf7002_eks.promiscuous:
+    build_only: true
+    extra_args: SHIELD=nrf7002ek
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpuapp
+    platform_allow: nrf5340dk/nrf5340/cpuapp
+    tags: ci_build
+  sample.nrf7000.promiscuous:
+    build_only: true
+    extra_args:
+      SHIELD=nrf7002ek_nrf7000
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpuapp
+    platform_allow: nrf5340dk/nrf5340/cpuapp
+    tags: ci_build
+  sample.nrf7001.promiscuous:
+    build_only: true
+    integration_platforms:
+      - nrf7002dk/nrf5340/cpuapp/nrf7001
+    platform_allow: nrf7002dk/nrf5340/cpuapp/nrf7001
+    tags: ci_build
```

Comment:
to minimise CI time, we can drop these as nothing specific to 7001/7000 is being done.